src/commands/classified/external.rs

```diff
@@ -49,23 +47,15 @@ impl Decoder for LinesCodec {
     }
 }
 
-#[derive(Debug)]
-pub(crate) enum StreamNext {
-    Last,
-    External,
-    Internal,
-}
-
 pub(crate) async fn run_external_command(
     command: ExternalCommand,
     context: &mut Context,
-    input: ClassifiedInputStream,
-    stream_next: StreamNext,
-) -> Result<ClassifiedInputStream, ShellError> {
-    let stdin = input.stdin;
-    let inputs: Vec<Value> = input.objects.into_vec().await;
-
+    input: InputStream,
+    is_last: bool,
```

Comment:
For now, I kept this here so tests pass. I could make it work with `autoview`, but some tests would fail because of all of the formatting (maybe autoview should understand if it's working with a TTY and not format?).